packages/webapp/__tests__/MainLayout.tsx

```diff
@@ -44,6 +46,6 @@ describe('MainLayout', () => {
     });
     const [el] = await screen.findAllByAltText(`idoshamun's profile`);
     expect(el).toHaveAttribute('src', 'https://daily.dev/ido.png');
-    expect(await screen.findAllByText('5')).toHaveLength(2);
```

Comment:
Why did we end up changing the evaluation here.

Reply:
I'll check this again, it started failing, not sure why it finds 2 in the first place so will dig a bit more.

Reply:
So the test without my changes runs on mobile, and because we have different things on mobile main layout, this starts failing in the new layout, it looks for an src where we don't show any. So test at line 46 fails, reason in this in mainLayoutHeader:
<img width="261" alt="Screenshot 2024-04-25 at 12 29 33 PM" src="https://github.com/dailydotdev/apps/assets/36197304/ea09134e-866d-4290-b0ec-0be01120e7b6">

If we force the test to run on laptop, like I've done now, the test fails even in production (that length 2 doesn't exist), so for a safer test until we cover both mobile and laptop, it's better to leave this run only on laptop, and fix the test for both production and here.

Reply:
Yeah, agreed. Since we removed legacy code, this would definitely be affected. Sounds good to me 🚀